src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -366,7 +367,7 @@ protected function loadFile($file)
         }
 
         try {
-            $configuration = $this->yamlParser->parse(file_get_contents($file));
+            $configuration = $this->yamlParser->parse(file_get_contents($file), Yaml::PARSE_CONSTANT);
         } catch (ParseException $e) {
```

Comment:
The `Symfony\Component\Yaml\Exception\RuntimeException` might not be caught here, @xabbuh any ideas on how we should handle it?